src/pages/SignUpPage.js

```diff
@@ -6,6 +6,9 @@ import { useSession, useDispatch } from 'hooks';
 import { signUp } from 'state/actions/userActions';
 import SignUpForm from 'components/user/SignUpForm';
 import routes from 'constants/routesPaths';
+import AuthWrapper from '../components/common/AuthWrapper';
+
+import './SignUpPage.scss';
```

Comment:
This import already exists below

Reply:
Huh, I apologize but I don't seem to see it. In which line is it duplicated?

Reply:
Hmm, I'm pretty sure it was there, it must've been removed at some point. Nevermind.